Devservices compose using oracle database construct wrong jdbc url

### Describe the bug

When using devservices compose with an oracle database using image gvenzl/oracle-free:23-slim-faststart the generated jdbc url is wrong and contains extra // like so : oracle:thin://@host:1521/db.

### Expected behavior

Devservices compose should generate the correct url for an oracle database like so : oracle:thin:@host:1521/db.

### Actual behavior

The generated jdbc or reactive url is wrong and the application cannot establish a connection.

Using quarkus:dev I get the error : 
Caused by: java.sql.SQLException: ORA-17067: Invalid Oracle URL specified

### How to Reproduce?

https://github.com/theCat69/quarkus-minimal-reproducer-oradb-dc

### Output of `uname -a` or `ver`

MSYS_NT-10.0-19045 DESKTOP-RSNRORB 3.5.4-0bc1222b.x86_64 2024-12-05 09:27 UTC x86_64 Msys

### Output of `java -version`

openjdk version "21.0.1" 2023-10-17 OpenJDK Runtime Environment (build 21.0.1+12-29) OpenJDK 64-Bit Server VM (build 21.0.1+12-29, mixed mode, sharing)

### Quarkus version or git rev

3.22.3

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.9.9

### Additional information

There is an issue in the code of OracleDatasourceServiceConfigurator class. The jdbc url and thus the reactive url are based on this string : "jdbc:%s://@%s:%d/%s%s". However it should be based on this string : "jdbc:%s:@%s:%d/%s%s".

--- a/docs/src/main/asciidoc/datasource.adoc
+++ b/docs/src/main/asciidoc/datasource.adoc
@@ -470,7 +470,7 @@ quarkus.datasource."pg".jdbc.url=jdbc:postgresql:///your_database
 
 quarkus.datasource."oracle".db-kind=oracle
 quarkus.datasource."oracle".active=false
-quarkus.datasource."oracle".jdbc.url=jdbc:oracle:///your_database
+quarkus.datasource."oracle".jdbc.url=jdbc:oracle:thin:@localhost:1521/your_database
 ----
 
 Setting `quarkus.datasource."pg".active=true` xref:config-reference.adoc#configuration-sources[at runtime] makes only the PostgreSQL datasource available.

--- a/extensions/devservices/oracle/src/main/java/io/quarkus/devservices/oracle/deployment/OracleDatasourceServiceConfigurator.java
+++ b/extensions/devservices/oracle/src/main/java/io/quarkus/devservices/oracle/deployment/OracleDatasourceServiceConfigurator.java
@@ -36,7 +36,7 @@ public class OracleDatasourceServiceConfigurator implements DatasourceServiceCon
     }
 
     public String getJdbcUrl(ContainerAddress containerAddress, String databaseName) {
-        return "jdbc:%s://@%s:%d/%s%s".formatted(
+        return "jdbc:%s:@%s:%d/%s%s".formatted(
                 getJdbcPrefix(),
                 containerAddress.getHost(),
                 containerAddress.getPort(),

